AgentDiscoveries-Frontend/app/src/components/admin/entity.jsx

```diff
@@ -22,8 +22,11 @@ export default class Entity extends React.Component {
     }
 
     getEntityRow() {
-        return Object.keys(this.props.entity).map(key =>
-            <td key={key}>{this.props.entity[key].toString()}</td>);
+        const headers = this.props.headers;
+        return headers.map((header) => {
+            if(header == "agentId" && this.props.entity[header] == undefined) this.props.entity[header] = "N/A";
```

Comment:
Do we only want the N/A rule to apply to `agentId`? Shouldn't any field that's missing display as an appropriate entry equivalently?

Reply:
Also, can we use === instead of == please? :)